components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/handler/transaction/TransactionConstants.java

```diff
@@ -18,18 +18,16 @@
 
 package org.wso2.micro.integrator.initializer.handler.transaction;
 
-import org.wso2.carbon.utils.ServerConstants;
-
-import java.io.File;
 
 public class TransactionConstants {
 
-    public static final String DEFAULT_SECURITY_RESOURCE_DIR_PATH = System.getProperty(ServerConstants.CARBON_HOME)
-            + File.separator + "repository" + File.separator + "resources" + File.separator + "security";
-    public static final String TRUSTSTORE_FILE = "wso2carbon.jks";
-
-    public static final String KEY_TYPE = "JKS";
-    public static final String KEY_ALIAS = "wso2carbon";
-    public static final String KEYSTORE_PASSWORD = "wso2carbon";
     public static final String ENCRYPTION_ALGORITHM = "RSA";
+
+    public static final String ENCRYPTION_PUB_KEY = "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuXUnLzRfEokhRQT62dnh\n"
```

Comment:
how was this generated. will this change if we change default certs

Reply:
How was this generated?
- The key pair was newly generated using OpenSSL for the use of this transaction count handler component

Will this change if we change default certs?
- No, because the public key is not embedded into any certificate (simply because we are not intending to do any secure communication or any such activities using this public key).